Source/Reporting/Web/src/app/Reporting/CaseReportsForListing/training-case-reports/training-case-reports.component.html

```diff
@@ -108,7 +119,11 @@ <h3>
                     {{caseReport.message}} {{ 'REGISTER.REPORT.parse_error' | translate }}: {{caseReport.parsingErrorMessage}}
                 </td>
             </ng-container>
-            <td><button (click)="convertToLiveCase(caseReport)" type="button" name="button" class="btn btn-warning">{{ 'REGISTER.REPORT.convert_to_live' | translate }}</button> </td>
+            <td>
+                <button (click)="convertToLiveCase(caseReport)" type="button" class="btn convert">
```

Comment:
Is the screenshot on the PR how it looks after the changes? If so I think this should look more like a button. If it already looks like a button you can ignore this comment :)

Reply:
Yeah, it should perhaps look more like a button :P